app/routes/questionnaire.py

```diff
@@ -179,22 +182,41 @@ def get_section(schema, questionnaire_store, section_id, list_item_id=None):
         questionnaire_store.metadata,
     )
 
+    if section_id not in router.enabled_section_ids:
+        return redirect(url_for(".get_questionnaire"))
+
+    location = Location(
+        section_id=section_id,
+        list_name=schema.get_repeating_list_for_section(section_id),
+        list_item_id=list_item_id,
+    )
+    section_summary = SectionSummary(schema, questionnaire_store, location, router)
```

Comment:
Now that we have a section summary handler, the router should be initialised by that and not accessed directly.